docs/sources/developers/plugins/_index.md

```diff
@@ -6,77 +6,48 @@ title: Build a plugin
 ---
 
-# Build a plugin
+# Welcome to the Grafana plugin developer's guide
 
-For more information on the types of plugins you can build, refer to the [Plugin Overview]({{< relref "../../administration/plugin-management/" >}}).
+Grafana is an open source software project for creating, exploring, and sharing all of your data through beautiful, flexible dashboards. Plugins extend its built-in functionality so that users can accomplish specialized tasks to make Grafana even more useful for them. 
 
-## Get started
+Many plugins are available in the [Catalog](https://grafana.com/grafana/plugins/) and others are built by developers for private use or to contribute them to the open source community. 
 
-The easiest way to start developing Grafana plugins is to use the Grafana [create-plugin tool](https://www.npmjs.com/package/@grafana/create-plugin).
+If you are a Grafana plugin developer or want to become one, this documentation is for you. Use it to get productive quickly, follow along with step-by-step tutorials, or you can refer back to it as you develop.  
 
-Open the terminal, and run the following command in your [plugin directory]({{< relref "../../setup-grafana/configure-grafana/#plugins" >}}):
+## Plugin basics
 
-```bash
-npx @grafana/create-plugin@latest
-```
+These are among the types of plugins you can create:
 
-Follow the questions and you will have a starter plugin ready to develop.
+- **Panel plugins** - Visualize data and navigating between dashboards.
+- **Data source plugins** - Link to new databases or other sources of data.
+- **App plugins** - Bundle panels and data sources together to create a better user experience.
 
-If you want a more guided introduction to plugin development, check out our tutorials:
+> **Note:** To learn more about the types of plugins you can build, refer to the [Plugin Overview]({{< relref "../../administration/plugin-management/" >}}) documentation.
 
-- [Build a panel plugin](/tutorials/build-a-panel-plugin/)
-- [Build a data source plugin](/tutorials/build-a-data-source-plugin/)
+## Contents of this developer's guide
 
-## Go further
-
-Learn more about specific areas of plugin development.
-
-### Tutorials
-
-If you're looking to build your first plugin, check out these introductory tutorials:
-
-- [Build a panel plugin](/tutorials/build-a-panel-plugin/)
-- [Build a data source plugin](/tutorials/build-a-data-source-plugin/)
-- [Build a data source backend plugin](/tutorials/build-a-data-source-backend-plugin/)
-
-Ready to learn more? Check out our other tutorials:
-
-- [Build a panel plugin with D3.js](/tutorials/build-a-panel-plugin-with-d3/)
+The following topics give you an idea of what to expect from this guide:
 
-### Guides
+- **[Get started with plugins](get-started-with-plugins/)** - The easiest way to start developing Grafana plugins is to use the [Create-plugin](https://www.npmjs.com/package/@grafana/create-plugin) tool. Use it to get your plugin up and running as fast as possible. 
```

Comment:
Removed the last sentence since it's duplicating content from the first sentence. 
```suggestion
- **[Get started with plugins](get-started-with-plugins/)** - The easiest way to start developing Grafana plugins is to use the [Create-plugin](https://www.npmjs.com/package/@grafana/create-plugin) tool.  
```

Reply:
will check in fix